README.md

```diff
@@ -287,7 +287,7 @@ appreciated and helps in maintaining the project.
 ## Maintainers
 
 In alphabetical order: [Carson Katri](https://github.com/carson-katri),
-[David Hunt](https://github.com/foscomputerservices),
+[David Hunt](https://github.com/foscomputerservices), [@ezraberch](https://github.com/ezraberch),
```

Comment:
```suggestion
[David Hunt](https://github.com/foscomputerservices), [Ezra Berch](https://github.com/ezraberch),
```

Reply:
Thanks, I thought this was your full name, but didn't want to make any assumptions without your approval 🙂